cli/azd/pkg/auth/manager.go

```diff
@@ -264,7 +264,7 @@ func newCredentialFromClientAssertion(
 }
 
 func (m *Manager) LoginInteractive(ctx context.Context, redirectPort int) (azcore.TokenCredential, error) {
-	options := []public.InteractiveAuthOption{}
+	options := []public.AcquireInteractiveOption{}
```

Comment:
What is the difference between these 2 types? `public.InteractiveAuthOption` and `public.AcquireInteractiveOption` ?

Reply:
It's the new name with the `azidentity` package update.